### DISCUSSION TITLE
Editor

### DISCUSSION THREAD

**Discussion by Geri-G93** [Courseplay/Courseplay_FS22#2970] [Ideas]
Hallo
der Ingame Editor ist soweit top, nur ConnectingTrack wegpunkte wären noch nötig

andere einfachere Idee wäre eine manuelle Kursaufzeichnung, gerade auf verwinkelten Feldern kommt der CourseGenerator an seine Grenzen und baut mist
mfg

**Comment by Tensuko** [comment]
Connection Track wird es nicht geben für den Editor.
Es wird einen neuen Generator geben in ferner Zukunft, aber es gibt eben auch Grenzen was die Geometrie von Feldern, den genutzten Tools mit Arbeitsbreite etc angeht.

Eine manuelle Kursaufzeichnung wird es auch nicht geben.

Wenn du statt "baut mist" einfach mal ein Beispiel hättest, könnte man auch mehr dazu sagen!
